### ISSUE TITLE
Incorrect custom unit in unit multiplication result

### ISSUE THREAD

**Issue by zcohan** [josdejong/mathjs#1286]
Here's my code:

```
math.createUnit('USD')
math.createUnit('EUR', {definition: '1.15 USD'})
math.eval('10 EUR/hour * 2 hours') //23 USD
```

I expected the result to be '20 EUR', but it's '23 USD'. I don't see a good reason why you'd automatically want the answer expressed in terms of the base unit, so I assume this is a bug.

**Comment by ericman314** [comment]
Thanks, there was indeed a bug. PR #1287 addresses this.

**Comment by josdejong** [comment]
This bug should be fixed now in `v5.2.2`. Thanks again @ericman314
